gateway/proxy/proxy/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "entrypoint.sh"
+
+iptables -t nat -N proxy_redirect
+iptables -t nat -A proxy_redirect -p tcp -d 104.18.29.94 -j REDIRECT --to-ports 8443
+iptables -t nat -A proxy_redirect -p tcp -d 104.18.28.94 -j REDIRECT --to-ports 8443
```

Comment:
assuming this is the IP for runtime? is this something we want to hardcode, or should it be configurable?

Reply:
@psschwei Thanks for review.
I hope that we can take put the iptables completely by setting the runtime url to the proxy.
Yes, I can change it configurable.  I have to try the dns name, too.

Reply:
It looks like runtime client has `proxies` field. https://github.com/Qiskit/qiskit-ibm-runtime/blob/c582e5789797861174bafcb96e6470f7b637d3b7/qiskit_ibm_runtime/qiskit_runtime_service.py#L77

Can we test our proxy with this field? 

If it will work we can ask runtime team to add ability to configure this field using env variables.

Reply:
Ah, disregard comment above. `proxies` parameter is for mapping between protocol and url

Reply:
Can we try to use `url` parameter and test if it will work? If it will work we can ask runtime team to add ability to configure this field using env variables.

https://github.com/Qiskit/qiskit-ibm-runtime/blob/c582e5789797861174bafcb96e6470f7b637d3b7/qiskit_ibm_runtime/qiskit_runtime_service.py#L73

Reply:
@IceKhan13 The `url` doesn't work.  It is used in the constructor of the `QiskitRuntimeService` to access the auth server.  The  `QiskitRuntimeService` is created at the client system instead of the ray node so if the url is pointing to the proxy, it fails in the constructor.